<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find percepcion_3d)/config/config.yaml" command="load"/>

  <include file="$(find percepcion_3d)/launch/gazebo.launch"/>

  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
    output="screen" ns="/my_robot" args="base_lidar_to_motor_joint_position_controller joint_state_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"
    respawn="false" output="screen">
    <remap from="/joint_states" to="/my_robot/joint_states" />
  </node>

  <node pkg="percepcion_3d" type="lidar_position_publisher.py" name="lidar_position_publisher" />

</launch>